invenio/b2share/modules/b2deposit/b2share_model/model.py

```diff
@@ -140,10 +139,8 @@ def __init__(self):
                            'were relevant in the creation of the resource.'
         }
         self.field_args['language'] = {
-            'hidden': True,
             'value': self.language_default,
-            # 'description':
-            # 'The name of the language the document is written in.'
+            'description': 'The name of the language the document is written in.'
```

Comment:
Agree, but this doesn't seem to be used, at least not visible, anywhere on the record page. So what's been the purpose of the field?